build-logic/src/main/kotlin/Mpp.kt

```diff
@@ -42,6 +38,8 @@ fun Project.configureMppDefaults(withJs: Boolean = true, withLinux: Boolean = tr
     )
 
     addTestDependencies(withJs)
+
+    enableNewMemoryManager()
```

Comment:
I think this one is useless. The new memory model is only used on binaries but all the modules calling into this should produce libraries, not binaries.

Reply:
Oh! Will try it.

Reply:
Yes you're totally right 😁